src/ServicePulse.Host/app/js/services/services.monitoring.js

```diff
@@ -16,6 +17,18 @@
 
             return function () {
                 clearInterval(interval);
+
+                // clear open toasts
+                for(var toastEntry in connectionToasts) {
```

Comment:
This seems like a UI specific concern being handled in the service, doesn't seem like it's in the right place